Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -85,17 +87,20 @@ public class PushDeploymentController : ApiController
                     Message = message
                 };
 
+                string remotebuild = _settings.GetValue(SettingsKeys.DoBuildDuringDeployment);
                 if (_settings.RunFromLocalZip())
                 {
-                    // This is used if the deployment is Run-From-Zip
-                    // the name of the deployed file in D:\home\data\SitePackages\{name}.zip is the
-                    // timestamp in the format yyyMMddHHmmss.
-                    deploymentInfo.ArtifactFileName = $"{DateTime.UtcNow.ToString("yyyyMMddHHmmss")}.zip";
-
-                    // This is also for Run-From-Zip where we need to extract the triggers
-                    // for post deployment sync triggers.
-                    deploymentInfo.SyncFunctionsTriggersPath = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
+                    SetRunFromZipDeploymentInfo(deploymentInfo);
+
+                    if (StringUtils.IsTrueLike(remotebuild)) deploymentInfo.DeploymentPath = "ZipDeploy. Run from package. Ignore remote build.";
+                    else deploymentInfo.DeploymentPath = "ZipDeploy. Run from package.";
```

Comment:
Nit: Parenthesis in the `if/else` will make code clearer to read.

Reply:
Added parenthesis for readability.